packages/react-ui-ag/src/ToggleDropDown/__tests__/ToggleDropDown-test.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import ThemedToggleDropDown from '../ToggleDropDown'
+import theme from './mocks/theme'
+
+const ToggleDropDown = ThemedToggleDropDown.WrappedComponent
+
+describe('ToggleDropDown', () => {
+  it('renders the base class and children', () => {
+    const props = {
+      theme,
+      className: theme.ToggleDropDown,
+      children: (<div>Content</div>),
+    }
+
+    const wrapper = mount(<ToggleDropDown {...props} />)
+    expect(wrapper.hasClass('ToggleDropDown')).toBeTruthy()
+    expect(wrapper.find('[data-tid="toggleDropDown-content"]').contains([<div>Content</div>])).toBeTruthy()
+  })
+
+  it('sets the default state to toggled on', () => {
+    const props = {
+      theme,
+      className: theme.ToggleDropDown,
+      children: <a href="#">Toggle</a>,
+    }
+
+    const wrapper = mount(<ToggleDropDown {...props} />)
+    expect(wrapper.find('[data-tid="toggleDropDown-button"]').first().hasClass('ToggleDropDown-Button-on')).toBeTruthy()
+  })
+
+  it('changes switches css class between ToggleDropDown-Button-on and ToggleDropDown-Button-off when toggled', () => {
```

Comment:
nitpick: typo - 'changes' and 'switches'

Reply:
Fixed